integr8ly/workload-web-app
# adding more metrics

JIRA:

https://issues.redhat.com/browse/DEL-266

To verify:

1. Check out the branch and run `make deploy`
3. Change the image of workload to use the `dev` tag rather than the `master` tag
2. Wait for a while and then check grafana (the RHMI one), there should be a new dashboard called `workload-web-app`. You should be able to view the dashboard and see some data.

Or check the one that's deployed here: https://grafana-route-redhat-rhmi-middleware-monitoring-operator.apps.weili.j0c0.s1.devshift.org/d/omfjL5mGk/workload-app?orgId=1&refresh=10s&from=now-3h&to=now. (ping me for credential)
